features/Terrain Shadows/Shaders/TerrainShadows/ShadowUpdate.cs.hlsl

```diff
@@ -1,13 +1,15 @@
 Texture2D<float> TexHeight : register(t0);
 RWTexture2D<float2> RWTexShadowHeights : register(u0);
 
-cbuffer ShadowUpdateCB : register(b1)
+cbuffer ShadowUpdateCB : register(b0)
```

Comment:
is b0 freed?

Reply:
Yes, deferredshared doesn't use it anymore